homeassistant/components/omnilogic/sensor.py

```diff
@@ -0,0 +1,484 @@
+"""Definition and setup of the Omnilogic Sensors for Home Assistant."""
+
+import logging
+
+from homeassistant.components.sensor import DEVICE_CLASS_TEMPERATURE
+from homeassistant.const import (
+    CONCENTRATION_PARTS_PER_MILLION,
+    MASS_GRAMS,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT,
+    VOLUME_LITERS,
+)
+
+from .common import OmniLogicEntity, OmniLogicUpdateCoordinator
+from .const import COORDINATOR, DOMAIN, PUMP_TYPES
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the sensor platform."""
+
+    sensor_types = {
```

Comment:
If we move the variable to the bottom of the module it can be a constant.

Reply:
Fought with that for longer than I should have... thanks. Addressed.